_layouts/versions.html

```diff
@@ -46,11 +48,13 @@
   - x64
   - arm64
   - x86
+  - all
```

Comment:
would `noarch` work better than all? I've not settled on this... 🤔

Reply:
`noarch` make me think "platform agnostic" (e.g. we use `noarch` to refer to FreeBSD packages that are identical on all architectures, e.g. python / ruby).  Here we indeed have platform-dependent packages because the application bundle architecture specific binaries (jna for OpenSearch; re2 for OpenSearch-Dashbaords).

Maybe `any` is a better fit?

Reply:
@smortex OK - so what architectures are supported in the FreeBSD version?

Reply:
@stockholmux we currently have i386, amd64 (x86, 32 and 64 bits) and aarch64 (arm 64 bits) packages available.  Other platforms might not have packages because of a failing dependency (I am not sure openjdk11 build on 32 bit ARM) or be delayed  because of limited build resources (I only saw aarch64 packages today).

Reply:
OK. That makes sense @smortex I think it makes sense to more explicit here (repeating near identical `/_artifact/` files) than less (using `all` or `any`). Two reasons:
1) I'd like these files to be useful to eventually to power an API and making sure there is an explicit entry for each architecture would simplify the the implementation.
2) Who knows, maybe someone ports OpenSearch to RISC-V (or whatever) and would be frustrated `all` not being true.

Reply:
Sure!  How should we name 32bit x86 (`i386` architecture with FreeBSD terminology)? `x86` (what Microsoft [seems to use for this architecture](https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/x86-architecture))?

Reply:
Replying to myself: `x86` is not listed in the _architectures_ list but appear in the _architecture_order_ so should be fine :-)